src/lib/ui/thermostat/ClimateSettingScheduleDevice.tsx

```diff
@@ -0,0 +1,54 @@
+import { isThermostatDevice, type ThermostatDevice } from 'seamapi'
+
+import { ChevronRightIcon } from 'lib/icons/ChevronRight.js'
+import { useDevice } from 'lib/seam/devices/use-device.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+export function ClimateSettingDevice({
+  deviceId,
+}: {
+  deviceId: string
+}): JSX.Element | null {
+  const { isLoading, device } = useDevice({
```

Comment:
I need to check on this but I think we will probably limit using the api hooks to the components in `lib/seam`. Instead this component should take an object with type compatible with a ThermostateDevice

Reply:
Interesting. What's the reasoning for that? It's likely that the device is already cached at this point.

Reply:
Might just be because of this / we haven't really implemented a component from full bottom up before https://github.com/seamapi/react/pull/378#discussion_r1308089157